doc/source/user_guide/howto.rst

```diff
@@ -0,0 +1,73 @@
+.. _how-to:
+
+.. currentmodule:: nomenclature
+
+This guide is intended as a quick instruction for adding a model to comparison project.
+It is still recommended to read the detailed instructions under :ref:`model_mapping` and
+:ref:`region` in particular.
+
+How to add your model mapping to a project repository
+=====================================================
+
+There are two parts to adding a new model mapping. The mapping itself and the easily
+forgotten additionally required region definitions. 
+
+As outlined in :ref:`model_mapping` a model mapping holds the information about model
+native an aggregation regions, for example:
```

Comment:
Region aggregation mapping
^^^^^^^^^^^^^^^^^^^^^^^^^^

As described in :ref:`model_mapping`, the model mapping holds the information about aggregating from
native regions to common regions. It also allows renaming of native regions for disambiguation in a
scenario ensemble.